xml/System.Text/Rune.xml

```diff
@@ -493,10 +491,18 @@ This method is very similar to <xref:System.Text.Rune.DecodeFromUtf8(System.Read
         <Parameter Name="value" Type="System.Text.Rune" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">The rune for which to get the numeric value.</param>
+        <summary>Gets the numeric value associated with <paramref name="value" />.</summary>
+        <returns>The numeric value associated with <paramref name="value" />, or -1 if <paramref name="value" /> does not represent a numeric character.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This method has behavior equivalent to <xref:System.Globalization.CharUnicodeInfo.GetNumericValue?displayProperty=nameWithType>. Refer to that documentation for more information.
```

Comment:
@mairaw @Jozkee all these xref with displayProperty failed to render, so the build finished with warnings. Should I rever them to %2A?

```
    [Warning] 1 invalid cross reference(s) "<xref:System.Char.IsLetterOrDigit?displayProperty=nameWithType>".
    [Warning] 1 invalid cross reference(s) "<xref:System.Char.IsLower?displayProperty=nameWithType>".
    [Warning] 1 invalid cross reference(s) "<xref:System.Char.IsNumber?displayProperty=nameWithType>".
    [Warning] 1 invalid cross reference(s) "<xref:System.Char.IsPunctuation?displayProperty=nameWithType>".
    [Warning] 1 invalid cross reference(s) "<xref:System.Char.IsSeparator?displayProperty=nameWithType>".
    [Warning] 1 invalid cross reference(s) "<xref:System.Char.IsSymbol?displayProperty=nameWithType>".
    [Warning] 1 invalid cross reference(s) "<xref:System.Char.IsUpper?displayProperty=nameWithType>".
    [Warning] 1 invalid cross reference(s) "<xref:System.Globalization.CharUnicodeInfo.GetNumericValue?displayProperty=nameWithType>".
    [Warning] 1 invalid cross reference(s) "<xref:System.Char.IsWhiteSpace?displayProperty=nameWithType>".
    [Warning] 1 invalid cross reference(s) "<xref:System.Globalization.CharUnicodeInfo.GetUnicodeCategory?displayProperty=nameWithType>".
    [Warning] 1 invalid cross reference(s) "<xref:System.Char.IsControl?displayProperty=nameWithType>".
    [Warning] 1 invalid cross reference(s) "<xref:System.Char.IsDigit?displayProperty=nameWithType>".
    [Warning] 1 invalid cross reference(s) "<xref:System.Char.IsLetter?displayProperty=nameWithType>".

```

Reply:
Example: https://review.docs.microsoft.com/en-us/dotnet/api/system.text.rune.isdigit?view=netcore-3.0&branch=pr-en-us-2876

Reply:
That was the change I did to add the %2A. Was that undone?

Reply:
I think I know what happened.
I left the suggestion of adding `?displayProperty=nameWithType` on a prior version to @mairaw 's commit (the one that adds `%2A` to the methods), then when my suggestions were applied, that `%2A` was lost.

I unintentionally did the review on the prev. version because when I started it, was the latest.

Reply:
No worries. I think I got all the conflicts resolved now. Please take a look.